src/components/EditMenu.vue

```diff
@@ -539,6 +546,32 @@
         </div>
       </div>
     </div>
+    <div
+      v-show="widgetMode === 'Custom'"
+      ref="availableCustomWidgetElementsContainer"
+      class="flex items-center w-full h-full gap-3 overflow-auto pr-2"
+    >
+      <div
+        v-for="element in availableCustomWidgetElementsTypes"
+        id="mini-widget-card"
+        :key="element.hash"
+        class="flex flex-col items-center justify-between rounded-md bg-[#273842] hover:brightness-125 h-[90%] aspect-square cursor-pointer elevation-4 overflow-clip pointer-events-none"
+        draggable="false"
+      >
+        <div />
+        <div id="draggable-mini-widget" class="m-2 pointer-events-auto select-auto cursor-grab" draggable="true">
+          <div class="pointer-events-none">
+            <CustomWidgetInstantiator :element="element" />
+          </div>
+        </div>
+        <div class="flex items-center justify-center w-full p-1 transition-all bg-[#3B78A8] rounded-b-md text-white">
+          <span class="whitespace-normal text-center">{{
+            element.name.replace(/([a-z])([A-Z])/g, '$1 $2').replace(/^./, (str) => str.toUpperCase()) ||
+            'Very Generic Indicator'
```

Comment:
Is this right?

Reply:
This code is turning a camelCase name such as RelativeAltitudeIndicator to Relative Altitude Indicator to fit the card title. But the exception is indeed unnecessary.